webnn/clamp.https.any.js

```diff
@@ -0,0 +1,17 @@
+// META: title=test WebNN API clamp operation
+// META: global=window,dedicatedworker
+// META: script=./resources/utils.js
+// META: timeout=long
+
+'use strict';
+
+// https://webmachinelearning.github.io/webnn/#api-mlgraphbuilder-clamp
+
+const computeClamp = (builder, resources) => {
+  // MLOperand clamp(MLOperand x, optional MLClampOptions options = {});
+  // use 'x' for input operand name
+  const [inputOperand] = createInputOperands(builder, resources, ['x']);
+  return builder.clamp(inputOperand, resources.options);
+};
+
+testWebNNOperation('clamp', '/webnn/resources/test_data/clamp.json', computeClamp);
```

Comment:
@huningxin, any suggestion for next adding tests converted from native tests, like NNAPI, DML, thanks. 
If **Solution2** was preferable, I would update `testWebNNOperation()` to support test by iterating each test file.
Solution 1: 
```js
testWebNNOperation('clamp', '/webnn/resources/test_data/clamp.json', computeClamp);
testWebNNOperation('clamp', '/webnn/resources/test_data/clamp_nnapi.json', computeClamp);
testWebNNOperation('clamp', '/webnn/resources/test_data/clamp_dml.json', computeClamp);
```
Solution 2:
```js
testWebNNOperation(
  'clamp',
  [
    '/webnn/resources/test_data/clamp.json', 
    '/webnn/resources/test_data/clamp_nnapi.json', 
    '/webnn/resources/test_data/clamp_dml.json'
  ],
  computeClamp);
```

Reply:
If we merged later converted NNAPI/DML tests into \<operation-name\>.json, not sperate tests by files (disadvantage: the file size of \<operation-name\>.json would be larger),we maybe optimize 
`testWebNNOperation()` 
to 
`testWebNNOperation([op-name1, op-name2...], func)`, like
`testWebNNOperation(['add', 'sub', 'mul', 'div', 'max', 'min', 'pow'], computeBinary)`

,any suggestions? Thanks.